server/sonar-web/src/main/js/components/issue/components/IssueTitleBar.js

```diff
@@ -66,10 +67,10 @@ export default function IssueTitleBar(props: Props) {
                   </span>
                 </li>}
               <li className="issue-meta">
-                <a
+                <Link
                   className="js-issue-permalink icon-link"
-                  href={getSingleIssueUrl(issue.key)}
-                  target="_blank"
+                  onClick={(e: Event) => e.stopPropagation()}
```

Comment:
We should create a `handleClick` method here or we will have a new function every time.
Maybe we should create helper methods to stop propagation of event and/or prevent default.